compare-teamsvecs/compare.py

```diff
@@ -0,0 +1,60 @@
+import argparse
```

Comment:
Hi @thangk  
please put this in the https://github.com/fani-lab/OpeNTF/tree/main/src/_msc folder.